components/Navbar.tsx

```diff
@@ -0,0 +1,278 @@
+  parentHref: string
+}
+
+const navigation = [
+  {
+    title: "Services",
+    href: "/services",
+    routes: [
+      { title: "Classroom", href: "/services/classroom" },
+      { title: "Computing", href: "/services/computing" },
+      {
+        title: "Campus File Storage and Transfer",
+        href: "/services/file-storage-and-transfer",
+      },
+      { title: "Rates", href: "/services/rates" },
+      { title: "Visualization Systems", href: "/services/visualization" },
+      { title: "Consulting", href: "/services/consulting" },
+    ],
+  },
+  {
+    title: "Documentation",
+    href: "/documentation",
+    routes: [
+      { title: "Oscar", href: "https://docs.ccv.brown.edu" },
+      {
+        title: "Stronghold",
+        href: "/services/computing#data-risk-level-3-computing-(stronghold)",
+      },
+      { title: "Globus", href: "/services/file-storage-and-transfer#globus" },
+      { title: "Jupyter Hub", href: "/services/classroom#jupyterhub" },
+    ],
+  },
+  {
+    title: "Our Work",
+    href: "/our-work",
+    routes: [
+      { title: "Collaborations", href: "#" },
+      {
+        title: "Software",
+        href: "/our-work/software",
+      },
+      {
+        title: "Workshops and Talks",
+        href: "/our-work/workshops-and-talks",
+      },
+      {
+        title: "Publications",
+        href: "https://publications.ccv.brown.edu",
+      },
+    ],
+  },
+  {
+    title: "Help",
+    href: "/help",
+    routes: [
+      { title: "Submit a Ticket", href: "mailto:support@ccv.brown.edu" },
+      {
+        title: "Discourse",
+        href: "https://ask.cyberinfrastructure.org/c/brown-research-computing/37",
+      },
+      {
+        title: "Slack",
+        href: "https://ccv-share.slack.com/signup#/",
+      },
+      {
+        title: "Office Hours",
+        href: "https://events.brown.edu/ccv/month",
+      },
+    ],
+  },
+  {
+    title: "About",
+    href: "/about",
+    routes: [
+      { title: "Mission", href: "/about#mission" },
+      {
+        title: "Office of Information Technology",
+        href: "/about#office-of-information-technology",
+      },
+      {
+        title: "Our Teams",
+        href: "/about#our-teams",
+      },
+      {
+        title: "People",
+        href: "/about#people",
+      },
+      {
+        title: "Opportunities",
+        href: "/about#opportunities",
+      },
+      {
+        title: "Events",
+        href: "https://events.brown.edu/ccv/month",
+      },
+      {
+        title: "Facilities Statement",
+        href: "/about#facilities-statement",
+      },
+      {
+        title: "Diversity Statement",
+        href: "/about#diversity-statement",
+      },
+    ],
+  },
+  {
+    title: "Blog",
+    href: "/blog",
+    routes: [],
+  },
+]
+
+const Navbar = () => {
+  return (
+    <div className="bg-neutral-50 p-2 shadow flex justify-between items-center">
+      {/* Logo Section */}
+      <section className="flex items-center space-x-3">
```

Comment:
`<section>` is a really confusing element for me... There's an argument that you should just be using a `<div>` here, but I'm not sure! Though, if you want to stick with the `<section>`, MDN says you should have a header for completeness.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section 

Curious to hear your thoughts!

Reply:
Truly no thoughts here except that it made it way easier for my brain to picture them as distinct segments when figuring out flexbox. I get confused with the "div div div div" nests. I'll revert to divs here -- they absolutely shouldn't have headings!